lib/elixir/pages/patterns-and-guards.md

```diff
@@ -415,6 +415,22 @@ Check.empty?({})
 #=> true
 ```
 
+### Using guards for map fields
+
+Guards also support the `map.field` syntax, this allows fine grained control on each field:
+
+```elixir
+def publish(review) when review.is_ready do
+  # ...
+  :ok
+end
+
+def fetch(query) when is_struct(query, Query) and query.limit > 0 do
+  # ...
+  :ok
+end
+```
```

Comment:
Nice catch!

I am not sure if this deserves its own section but we can add an entry to the list in `## Guards`. WDYT?

Reply:
Sounds good, I've added an entry there 👌